scene/main/viewport.cpp

```diff
@@ -3781,13 +3769,23 @@ Viewport::Viewport() {
 	unhandled_key_input_group = "_vp_unhandled_key_input" + id;
 
 	// Window tooltip.
-	gui.tooltip_delay = GLOBAL_DEF("gui/timers/tooltip_delay_sec", 0.5);
+	float tooltip_delay = GLOBAL_DEF("gui/timers/tooltip_delay_sec", 0.5);
 	ProjectSettings::get_singleton()->set_custom_property_info("gui/timers/tooltip_delay_sec", PropertyInfo(Variant::FLOAT, "gui/timers/tooltip_delay_sec", PROPERTY_HINT_RANGE, "0,5,0.01,or_greater")); // No negative numbers
 
+	gui.tooltip_timer = memnew(Timer);
+	gui.tooltip_timer->set_wait_time(tooltip_delay);
+	gui.tooltip_timer->set_one_shot(true);
+	gui.tooltip_timer->connect("timeout", callable_mp(this, &Viewport::_gui_show_tooltip));
+	add_child(gui.tooltip_timer);
+
 	set_sdf_oversize(sdf_oversize); //set to server
 }
 
 Viewport::~Viewport() {
+	if (gui.tooltip_timer) {
+		gui.tooltip_timer->queue_delete();
```

Comment:
This node is already child of viewport and doesn't need to be deleted(this line cause invalid read in CI - https://github.com/godotengine/godot/runs/3144902320?check_suite_focus=true)

Reply:
This code is no longer relevant, see #43693

Reply:
Yup, rebase without the changes, but still good to know for other cases :)